pkg/helm/actions/setup_test.go

```
@@ -175,13 +188,28 @@ func setupTestOCIBasicAuth() error {
+		return fmt.Errorf("zot basic-auth not ready: %w", err)
+	}
 	if err := ExecuteScript("./testdata/uploadOciCharts.sh", true, "--basic-auth"); err != nil {
 		return err
 	}
 	return nil
 }
 
+func waitForTCP(addr string, timeout time.Duration) error {
+	deadline := time.Now().Add(timeout)
+	for time.Now().Before(deadline) {
+		conn, err := net.DialTimeout("tcp", addr, time.Second)
```

golangci-lint (2.12.2)

[error] net.DialTimeout must not be called. use (*net.Dialer).DialContext with (*net.Dialer).Timeout

(noctx)